src/librustdoc/clean/auto_trait.rs

```diff
@@ -438,7 +424,7 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
     // * We explicitly add a '?Sized' bound if we didn't find any 'Sized' predicates for a type
     fn param_env_to_generics(
         &self,
-        tcx: TyCtxt<'tcx>,
+        cx: &mut DocContext<'tcx>,
```

Comment:
Yes: https://github.com/rust-lang/rust/pull/82020#discussion_r575917209

Reply:
Here's the error if I take `&self` *and* `&mut DocContext`:

```
error[E0502]: cannot borrow `self.cx` as mutable because it is also borrowed as immutable
  --> src/librustdoc/clean/auto_trait.rs:71:29
   |
70 |                         let new_generics = self.param_env_to_generics(
   |                                            ---- --------------------- immutable borrow later used by call
   |                                            |
   |                                            immutable borrow occurs here
71 |                             &mut self.cx,
   |                             ^^^^^^^^^^^^ mutable borrow occurs here

error[E0500]: closure requires unique access to `self` but it is already borrowed
  --> src/librustdoc/clean/auto_trait.rs:56:82
   |
56 |                     self.f.find_auto_trait_generics(ty, param_env, trait_def_id, |infcx, info| {
   |                     ------ ------------------------                              ^^^^^^^^^^^^^ closure construction occurs here
   |                     |      |
   |                     |      first borrow later used by call
   |                     borrow occurs here
...
70 |                         let new_generics = self.param_env_to_generics(
   |                                            ---- second borrow occurs due to use of `self` in closure
```

Reply:
This has since been removed now that I figured out I was being dumb :)